include/field_map_obj.h

```diff
@@ -46,7 +46,7 @@ void FieldObjectClearAnim(struct MapObject *);
 void FieldObjectClearAnimIfSpecialAnimActive(struct MapObject *);
 void SpawnFieldObjectsInView(s16, s16);
 u8 sprite_new(u8, u8, s16, s16, u8, u8);
-u8 AddPseudoFieldObject(u16, void (*)(struct Sprite *), s16, s16, u8);
+u8 AddPseudoEventObject(u16, void (*)(struct Sprite *), s16, s16, u8);
```

Comment:
Suggest using the type [`SpriteCallback`](include/sprite.h#L172) for the second argument